packages/scm/src/browser/scm-widget.tsx

```diff
@@ -90,6 +90,11 @@ export class ScmWidget extends BaseWidget implements StatefulWidget {
         return this.panel.layout as PanelLayout;
     }
 
+    collapseAll(): void {
```

Comment:
please remove, we should follow unidirectional flow: we update state, widget listens to state and rerender. We don't random updates of views here and there.